### PR TITLE
prevent too many genesis council members

### PR DESCRIPTION
**Pull request by kianenigma**
An insubstantial check that was overlooked.

### CODE REVIEW

File: frame/elections-phragmen/src/lib.rs [hunk 1]
```diff
@@ -213,36 +213,42 @@ decl_storage! {
...
-					vec![member.clone()],
-					*stake,
-				).expect("Genesis member could not vote.");
-
-				member.clone()
-			}).collect::<Vec<T::AccountId>>();
+			assert!(
+				config.members.len() as u32 <= T::DesiredMembers::get(),
+				"Cannot accept more than DesiredMembers genesis member",
+			);
```
Comment (thiolliere) [inline frame/elections-phragmen/src/lib.rs:219]: this is the only change isn't it ?

**Review by thiolliere — APPROVED** [review]
looks good to me,
I actually prefer the old formatting :-)

File: frame/elections-phragmen/src/lib.rs (hunk 1 above)
Comment (kianenigma) [reply frame/elections-phragmen/src/lib.rs:219]: yes, and I will revert the formatting. it is annoying that now it seems like I changed it all.

**Comment by thiolliere** [comment]
bot merge

**Review by shawntabrizi — APPROVED** [review]

**Comment by shawntabrizi** [comment]
bot merge

**Pull request merged**